pages/pipelines/group_step.md.erb

```diff
@@ -158,7 +158,7 @@ steps:
 
 ## Group merging
 
-If you upload a pipeline that has a `group` or `label` already present in your YAML configuration file, the groups will be merged together in the Buildkite UI.
+If you upload a pipeline from another pipeline whose `group` or `label` same as the current pipeline, then groups will be merged together in the Buildkite UI.
```

Comment:
```suggestion
If you upload a pipeline that has a `group` or `label` that matches the `group` or `label` in your YAML configuration file, the groups will be merged together in the Buildkite UI.
```

Reply:
Here is a suggestion that *merges* the two suggestions ;-)

```suggestion
If you upload a pipeline that has a `group` or `label` that matches a `group` or `label` in the current pipeline, those groups will be merged together in the Buildkite UI.
```

Is it only in the UI that the groups are merged? Or are they merged, *and* shown as merged in the UI